adapters/repos/db/aggregator/unfiltered_type_specific.go

```diff
@@ -273,8 +307,7 @@ func (ua unfilteredAggregator) parseAndAddDateRowSet(agg *dateAggregator, k []by
 ) error {
 	if len(k) != 8 {
 		// dates are stored as epoch nanoseconds, we expect to see an int64
-		return fmt.Errorf("unexpected key length on inverted index, "+
-			"expected 8: got %d", len(k))
+		return fmt.Errorf("parseAndAddDateRowSet: unexpected key length on inverted index, expected 8: got %d", len(k))
```

Comment:
I think that we should not expose our underlying API in error messages, cane we remove `parseAndAddDateRowSet`?